packages/api/src/brevo-contact/brevo-contacts.service.ts

```diff
@@ -128,10 +128,6 @@ export class BrevoContactsService {
                 if (contactIsInTargetGroupByAttributes) {
                     targetGroupIds.push(targetGroup.brevoId);
                 }
-
-                if (targetGroup.assignedContactsTargetGroupBrevoId && contact?.listIds.includes(targetGroup.assignedContactsTargetGroupBrevoId)) {
-                    targetGroupIds.push(targetGroup.brevoId, targetGroup.assignedContactsTargetGroupBrevoId);
-                }
```

Comment:
Why did you remove this? The manually assigned contact groups do not get added anymore if you remove this